main.tf

```diff
@@ -90,3 +90,26 @@ data "aws_acm_certificate" "cert" {
   // TODO the trim function would have been preferred, but is not available with terraform 0.12.16, fix will be available in 0.12.17
   domain = replace(data.aws_route53_zone.zone.name, "/\\.$/", "")
 }
+
+// Security Group info
+data "aws_security_group" "ssh_rdp" {
+  filter {
+    name = "vpc-id"
+    values = [data.aws_vpc.vpc.id]
+  }
+  filter {
+    name = "group-name"
+    values = ["*ssh*"]
+  }
+}
+
+data "aws_security_group" "rds" {
+  filter {
+    name = "vpc-id"
+    values = [data.aws_vpc.vpc.id]
+  }
+  filter {
+    name = "group-name"
+    values = ["*rds*"]
+  }
+}
```

Comment:
Seems like we're just hoping we don't have naming conflicts here. If somebody creates another security group that has "ssh" or "rds" in the name, won't this blow up (or at least behave unpredictably)?

If these security groups are being created by ACS in all accounts, we should have ACS add SSM Parameters for these like it does for the other things it creates.

Reply:
True. They are created by ACS. How can we get them added as SSM parameters?

Reply:
https://github.com/byu-oit/aws-acs/issues

Reply:
I've opened an issue: https://github.com/byu-oit/aws-acs/issues/97

Reply:
I guess what I am suggesting here is an internal optimization that doesn't affect the contract of this module. I'm fine if we move forward with things the way you've written them, and refactor after the enhancement to ACS is done.